changelog.txt

```diff
@@ -1,6 +1,7 @@
 *** Changelog ***
 
 = 10.2.0 - xxxx-xx-xx =
+* Dev - Removes the merchant email address from the System Status Report
```

Comment:
I have two minor comments:
1. I think this should qualify as a Fix rather than a Dev change. This does have implications for merchants.
2. I think it would be a bit clearer if we use "Remove" instead of "Removes", as that matches the approach for most of our changelog entries.
```suggestion
* Dev - Remove the merchant email address from the System Status Report
```